### PR TITLE
Update scalardb, scalardl and scalardl-audit chart for grafana

### PR DESCRIPTION
**Pull request by miseyu**
# Summary
- Fixed the dashboard because the `scalardb` metrics name changed.
- Changed datasource name to `Prometheus`.
- Add and fix missing manifest for metrics.

# Related
- https://github.com/scalar-labs/scalardb/pull/274

### CODE REVIEW

**Comment by feeblefakie** [comment]
@miseyu Thank you!
Since it's a public repo and Helm/K8s-related PRs will be reviewed by external reviewers (like superbrothers),
it would be great if you don't refer to Jira links and explain the PR. (possibly with some backgrounds)

**Comment by miseyu** [comment]
@feeblefakie 
Thank you for your understanding.
From now on, we will try to describe the details of the Public repository without referring to the Jira link.
I apologize for the inconvenience. :bow:

Thanks also for ` assignees ` and ` label `.
We will try to set them in the future.

**Comment by feeblefakie** [comment]
@miseyu Thank you! (and sorry for not having really explained it 🙇 )

**Comment by superbrothers** [comment]
> Changed datasource name to Prometheus.

Has the default name been changed?

**Review by superbrothers — APPROVED** [review]
LGTM

**Comment by miseyu** [comment]
@superbrothers 
Thank you very much for your confirmation. :bow:
The default was set to `Prometheus`, so this is a change to match the default.
https://github.com/scalar-labs/helm-charts/pull/24/files#diff-d9909689e538fc7e3038fdde6f743657876a15d1fe4bd338ac666c7242e8aabdR106

Sorry for the confusion. :bow:

**Review by ymorimo — APPROVED** [review]
LGTM!

File: charts/scalardb/files/scalardb_grafana_dashboard.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
       "targets": [
```
Comment (feeblefakie) [inline charts/scalardb/files/scalardb_grafana_dashboard.json:79]: These changes about Scalar DB dashboard is expected?  (I was expecting the changes in charts of scalardl and scalardl-audit from the PR title)
If it is not related, it would be great if this is a separate PR. (the changes look good!)

**Review by feeblefakie — COMMENTED** [review]
Looking good! Thank you!
I left one clarifying question.

File: charts/scalardb/files/scalardb_grafana_dashboard.json (hunk 1 above)
Comment (miseyu) [reply charts/scalardb/files/scalardb_grafana_dashboard.json:79]: @feeblefakie 
Sorry.
I had mentioned it in the description.
```
Fixed the dashboard because the scalardb metrics name changed.
```

The title was confusing, so I have corrected the title.
https://github.com/scalar-labs/helm-charts/pull/24#event-5269350612

Comment (feeblefakie) [reply charts/scalardb/files/scalardb_grafana_dashboard.json:79]: @miseyu Oh, got it. Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
